src/Signer/Ecdsa/SignatureSerializer.php

```diff
@@ -1,72 +0,0 @@
-<?php
-declare(strict_types=1);
-
-namespace Lcobucci\JWT\Signer\Ecdsa;
-
-use GMP;
-use Mdanter\Ecc\Crypto\Signature\Signature;
-use Mdanter\Ecc\Crypto\Signature\SignatureInterface;
-use Mdanter\Ecc\Math\GmpMathInterface;
-use const STR_PAD_LEFT;
-use function gmp_init;
-use function pack;
-use function sprintf;
-use function str_pad;
-use function str_split;
-use function unpack;
-
-class SignatureSerializer
```

Comment:
BC break to be documented

Reply:
This method was always considered as internal, but we were not using `@internal` yet. I'm not sure if this really need to be stated as BC-break :(